### DISCUSSION TITLE
Access to local network

### DISCUSSION THREAD

**Discussion by BilboBaggins2** [bubuntux/nordvpn#126] [Q&A]
Hello, I would like to know if the vpn must be up and running in order to access the different containers through nginx. I suppose those are working independently. I am doing first some testing before subscribing to NordVPN. Maybe my settings are not correct.

**Comment by bubuntux** [comment]
I move this into a discussion, since is not really an issue.

yes the vpn needs to be up since is the container that provides the network for all the other containers, please follow the instructions on the README, if you run into a problem specify what you tried and what didn't work.

**Comment by BilboBaggins2** [comment]
Ok great thanks
